playbooks/common/openshift-cluster/initialize_openshift_version.yml

```diff
@@ -18,12 +18,17 @@
       msg: Incompatible versions of yum and subscription-manager found. You may need to update yum and yum-utils.
     when: "not openshift.common.is_atomic | bool and 'Plugin \"search-disabled-repos\" requires API 2.7. Supported API is 2.6.' in yum_ver_test.stdout"
 
+# TODO(jchaloup): find a different way how to make repoquery --qf '%version` atomic-openshift work without disabling the excluders
```

Comment:
maybe like this

mkdir /etc/openshift-ansible
cat /etc/yum.conf | grep -v 'excludes=' > /etc/openshift-ansible/yum.conf
repoquery --conf /etc/openshift-ansible/yum.conf